### PR TITLE
Bug 1166725 - Gaia UI tests that make or receive phone calls need to have external = true in their manifest.ini

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=595645182ce87ab870258d97ad745a036230c163

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3448a412a9b3738a2fb7c2bdbda5eb8d2c0f865f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bb0a8e82409095958426b9ee5e1c395b0b1bc61e

File: tests/python/gaia-ui-tests/gaiatest/tests/manifest.ini [hunk 1]
```diff
@@ -47,7 +47,7 @@ smoketest = false
 # Is a sanity test for core device features
 sanity = false
 
-# The test requires the use of external/internet resources
+# The test requires the use of internet resources or the Plivo server
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/manifest.ini]: FYI for future potential reader, we send HTTP requests to Plivo to trigger the phone calls. That's why Plivo is considered as an external internet resource.

**Pull request closed without merging**